datalad/downloaders/s3.py

```diff
@@ -46,10 +46,21 @@ class S3Authenticator(Authenticator):
     allows_anonymous = True
     DEFAULT_CREDENTIAL_TYPE = 'aws-s3'
 
-    def __init__(self, *args, **kwargs):
-        super(S3Authenticator, self).__init__(*args, **kwargs)
+    def __init__(self, host=None, **kwargs):
```

Comment:
Why do the unrelated drop of `*args` here?  While it probably doesn't matter, I don't see any reason to make that backwards-incompatible change.

Reply:
fair enough -- will RF to use `kwargs.pop`. In the longer run -- the whole thing could be RFed to become more explicit/consistent.

Reply:
> will RF to use `kwargs.pop`

All right, though keeping `*args` around doesn't require doing that.

Reply:
ah, right -- python3 !

Reply:
ok, this one is done